recipes/libsolace/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class SolaceTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is
+        # in "test_package"
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
```

Comment:
Remove imports, it's no longer required since you are using `run_environment` argument in `self.run`

Reply:
done